assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,34 +1,34 @@
 ﻿<Root>
     <File Name="object_kin2_obj" Segment="6">
         <DList Name="object_kin2_obj_DL_000120" Offset="0x120" />
-        <DList Name="object_kin2_obj_DL_000128" Offset="0x128" />
-        <DList Name="object_kin2_obj_DL_0002C0" Offset="0x2C0" />
-        <DList Name="object_kin2_obj_DL_000360" Offset="0x360" />
-        <Collision Name="object_kin2_obj_Colheader_000490" Offset="0x490" />
+        <DList Name="gOceanSpiderHouseDebrisDL" Offset="0x128" />
+        <DList Name="gOceanSpiderHouseBombableWallCrackDL" Offset="0x2C0" />
+        <DList Name="gOceanSpiderHouseBombableWallDL" Offset="0x360" />
+        <Collision Name="gOceanSpiderHouseBombableWallColHeader" Offset="0x490" />
         <DList Name="object_kin2_obj_DL_000650" Offset="0x650" />
         <DList Name="object_kin2_obj_DL_000658" Offset="0x658" />
         <Collision Name="object_kin2_obj_Colheader_000798" Offset="0x798" />
         <DList Name="object_kin2_obj_DL_000820" Offset="0x820" />
-        <DList Name="object_kin2_obj_DL_000828" Offset="0x828" />
+        <DList Name="gOceanSpiderHouseFireplaceGrateDL" Offset="0x828" />
         <Collision Name="object_kin2_obj_Colheader_000908" Offset="0x908" />
         <DList Name="object_kin2_obj_DL_000CA0" Offset="0xCA0" />
-        <DList Name="object_kin2_obj_DL_000CA8" Offset="0xCA8" />
-        <Collision Name="object_kin2_obj_Colheader_000F80" Offset="0xF80" />
+        <DList Name="gOceanSpiderHouseBookshelfDL" Offset="0xCA8" />
+        <Collision Name="gOceanSpiderHouseBookshelfColHeader" Offset="0xF80" />
         <DList Name="object_kin2_obj_DL_0010F0" Offset="0x10F0" />
-        <DList Name="object_kin2_obj_DL_0010F8" Offset="0x10F8" />
-        <Collision Name="object_kin2_obj_Colheader_001328" Offset="0x1328" />
+        <DList Name="gOceanSpiderHouseNightstandDL" Offset="0x10F8" />
+        <Collision Name="gOceanSpiderHouseNightstandColHeader" Offset="0x1328" />
```

Comment:
I'd say it's more a small chest of drawers since it's not associated to a bed in its use in-game.

Reply:
Just pushed the idea into the PR. Thank you!